release/release-trueos.sh

```diff
@@ -76,54 +77,54 @@ setup_poudriere_conf()
 		| grep -v "GIT_PORTSURL=" \
 		| grep -v "USE_TMPFS=" \
 		| grep -v "BASEFS=" \
-		> ${POUDRIERE_ETCDIR}/poudriere.conf
+		> ${_pdconf}
 	echo "Using zpool: $ZPOOL"
-	echo "ZPOOL=$ZPOOL" >> ${POUDRIERE_ETCDIR}/poudriere.conf
+	echo "ZPOOL=$ZPOOL" >> ${_pdconf}
 	echo "Using Dist Directory: $DIST_DIR"
-	echo "FREEBSD_HOST=file://${DIST_DIR}" >> ${POUDRIERE_ETCDIR}/poudriere.conf
+	echo "FREEBSD_HOST=file://${DIST_DIR}" >> ${_pdconf}
 	echo "Using Ports Tree: $GH_PORTS"
-	echo "GIT_URL=${GH_PORTS}" >> ${POUDRIERE_ETCDIR}/poudriere.conf
-	echo "USE_TMPFS=data" >> ${POUDRIERE_ETCDIR}/poudriere.conf
-	echo "BASEFS=$POUDRIERE_BASEFS" >> ${POUDRIERE_ETCDIR}/poudriere.conf
-	echo "ATOMIC_PACKAGE_REPOSITORY=no" >> ${POUDRIERE_ETCDIR}/poudriere.conf
-	echo "PKG_REPO_FROM_HOST=yes" >> ${POUDRIERE_ETCDIR}/poudriere.conf
+	echo "GIT_URL=${GH_PORTS}" >> ${_pdconf}
+	echo "USE_TMPFS=data" >> ${_pdconf}
+	echo "BASEFS=$POUDRIERE_BASEFS" >> ${_pdconf}
+	echo "ATOMIC_PACKAGE_REPOSITORY=no" >> ${_pdconf}
+	echo "PKG_REPO_FROM_HOST=yes" >> ${_pdconf}
 
```

Comment:
We might want to add ALLOW_MAKE_JOBS_PACKAGES="pkg ccache py* gcc* webkit* llvm* clang* perl* firefox* ruby* cmake* binutils p5* php*" to speed up the build.

Reply:
Agreed, but should that be here, or in /etc/poudriere.conf.release so you can customize it on an individual build node?

Reply:
Ok, since that can be overridden I've gone ahead and added it.